src/python/pypresso.cmakein

```diff
@@ -8,9 +8,9 @@
 #
 
 if test -n "$PYTHONPATH"; then
-  PYTHONPATH=@PYTHON_DIR@:$PYTHONPATH
+  PYTHONPATH="@PYTHON_DIR@:$PYTHONPATH"
 else
-  PYTHONPATH=@PYTHON_DIR@
+  PYTHONPATH="@PYTHON_DIR@"
 fi
```

Comment:
Just use
```
PYTHONPATH="@PYTHON_DIR@${PYTHONPATH:+:}$PYTHONPATH"
```
instead.

Reply:
Cryptic, but posix.

Reply:
In #2932, you used `${var+x}`, not `${var:+x}`, which makes a huge different, as it doesn't check for empty but defined variables.

Looking at #2932, you really need to run the bashism script on the scripts, that don't explicit use `bash` (or change them all to `bash`) and in the `bash` script, I would always use `[[  ]]` instead of `[ ]` as it automatically quotes the left hand side. And using brace-quotes don't replace double quotes unfortunately.

Reply:
I had plans to reduce bashisms (#3242), but there is more interest in our team to actually move away from shell scripts entirely ([ES meeting Oct 16](https://github.com/espressomd/espresso/wiki/Espresso-meeting-2019-10-16#code-quality-and-linting)).

Reply:
You guys should enable codacy, that helped us a lot to incrementally improve the shell scripts.

Reply:
I did not spot the difference between `${var+x}` and `${var:+x}` until you pointed it out to me. This is an issue inherent to shell. I don't mind using advanced variable substitution tricks in my personal shell scripts, but it's a different story on collaborative project. We aim to lower the entry barrier for new contributors on espresso, and this is an example where a simple if/else is more explicit than a not well-known variable substitution, in my opinion.

We used to have automatic code linting and checking with `pylint` and `CodeFactor` in the past, but they were eventually removed from CI a year ago. This is something I would like to experiment with again in 4.2.

Reply:
https://app.codacy.com/manual/junghans/espresso/dashboard